### ISSUE TITLE
new Mondo files for rare disease drug repurposing

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/mondo#7849]
We need to create some mondo subset files for use in KGs along the following lines:

- Rare disease subset
- Rare disease subset with leaf nodes representing variants in same gene rolled up to parent term
- All genetic diseases
- Rare disease subset where all diseases known to be related to the same Reactome or KEGG pathway or RHEA reaction are annotated as such  
- All diseases without known treatments (stretch goal)

This will require querying the MOnarch KG and/or other KGs, but the idea would be to release these files as part of Mondo release package moving forward. This ticket is intended as a starting place, feedback welcome.

**Comment by matentzn** [comment]
These are a lot of different subsets. Before we think about actually making specific files, lets make sure we understand the use cases correctly:

> Rare disease subset with leaf nodes representing variants in same gene rolled up to parent term

I am assuming @sabrinatoro can explain what this exactly means when she is back. 

- leaf nodes representing variants (you mean, diseases that are variant-specific?)
- You want to propagate variant associations from a child disease up to its parent? That is an annotation roll up and does not belong in an ontology (due to its all-some semantics). Seems you are asking for a KG extract here?

> Rare disease subset where all diseases known to be related to the same Reactome or KEGG pathway or RHEA reaction are annotated as such

Simple KG query, not suitable as a Mondo subset I think - this could be delivered as a KG component if it has value to be distributed in isolation.

I think this ticket should be moved to the monarch-kg repo, but we can leave it hear to shepherd it for the time being.

**Comment by mellybelly** [comment]
RE first question: leaf nodes that are subclasses of a disease with a single gene linked, and the subclasses are the variants in that same gene. 

As to where to put the ticket, we need the files released as mondo files, so the task is both KG and mondo release processes, happy to move or split the ticket though.

**Comment by twhetzel** [comment]
Since Mondo only includes gene annotations on diseases when caused by a single gene, will any additional modeling be added into Mondo to include gene annotations for diseases where >1 gene is involved?

**Comment by mellybelly** [comment]
> Since Mondo only includes gene annotations on diseases when caused by a single gene, will any additional modeling be added into Mondo to include gene annotations for diseases where >1 gene is involved?

this is in the monarch KG, not in mondo at present. this is why its confusing about where this ticket should be :-).

**Comment by monicacecilia** [comment]
Yes, this ticket belongs in monarch-app, as it requests querying the KG. I'll transfer it. 
The outputs of the KG query can be disseminated with the Mondo monthly releases. We should make a separate ticket about it in the Mondo repo. @matentzn, can you please do that?
